internal/provider/resource_consul_cluster.go

```diff
@@ -135,6 +117,16 @@ func resourceConsulCluster() *schema.Resource {
 				Optional:    true,
 				ForceNew:    true,
 			},
+			"size": {
+				Description:      "The t-shirt size representation of each server VM that this Consul cluster\nis provisioned with. Valid options - x_small, small, medium, large.",
```

Comment:
Trying to think of how we could ensure this description stays up-to-date with the enum values, but it looks like the SDK doesn't provide them in a list, just constants. 🤔

Reply:
Yea this was the other place I did not like the hard-coding :'( The compromise was at least the sdk validation will be our gate check. But, open to suggestions if any.